algorithmia/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+duration_milliseconds,gauge,,millisecond,,Duration of algorithm run,0,algorithmia,
+<USER-DEFINED>,gauge,,,,Inference-related metrics defined by an algorithm developer,0,algorithmia,
```

Comment:
We'll need this to be a specific / consistently named set of metrics.

Reply:
That makes sense, thanks! I removed the `<USER-DEFINED>` row from `metadata.csv` and instead clarified the point about additional user-defined metrics in the README.